Makefile

```diff
@@ -37,7 +37,7 @@ guest/.tinygo-target.json: scheduler/go.mod
 build-wat: $(wildcard scheduler/test/testdata/*/*.wat)
 	@for f in $^; do \
         wasm=$$(echo $$f | sed -e 's/\.wat/\.wasm/'); \
-		wat2wasm -o $$wasm --debug-names $$f; \
```

Comment:
When I ran original `make build-wat` command, I encountered the error: `The 'path' argument must be of type string. Received undefined.`
Is this command correct?

Reply:
Seems like CI works fine. So maybe this is because I am using macOS